### DISCUSSION TITLE
Search by daft_url?

### DISCUSSION THREAD

**Discussion by johnpaulhayes** [AnthonyBloomer/daftlistings#156] [Q&A]
Hi 👋 , 

Great library by the way! I'm really impressed by it. 

Is it possible to search by daft_url? 
If not, how could this be done (thinking of implementing...)

--
Thanks!
JP

**Comment by sdfordham** [comment]
This is not possible right now because this package generates a json and POSTs to a daft endpoint, it doesn't generate a URL. So this would amount to writing a converter from url to payload, and robustness may take some time e.g. sometimes the location is the last element of the daft url path but if more than one location is selected then it goes in the url query. The url query parameters do not look sufficiently generic so you would possibly have to write a parser for each url query parameter to figure out the relevant `Daft` class method.
